azure-iot-hub/azure/iot/hub/protocol/operations/configuration_operations.py

```diff
@@ -50,19 +51,19 @@ def get(self, id, custom_headers=None, raw=False, **operation_config):
          :class:`HttpOperationError<msrest.exceptions.HttpOperationError>`
         """
         # Construct URL
-        url = self.get.metadata["url"]
-        path_format_arguments = {"id": self._serialize.url("id", id, "str")}
+        url = self.get.metadata['url']
+        path_format_arguments = {
+            'id': self._serialize.url("id", id, 'str')
```

Comment:
>"id" [](start = 38, length = 4)

Not sure if this matters or not, but to keep it consistent use ' here instead of "

Reply:
Normalize these if necessary

---
In reply to: [388524263](https://github.com/Azure/azure-iot-sdk-python/pull/479#discussion_r388524263) [](ancestors = 388524263)